www/www-deploy.yml

```diff
@@ -137,7 +137,7 @@
     # support
     - match: "~/site/support/?$"
       dest: /docs
-    - match: "~/site/support/ome-artwork/?$"
+    - match: "~/site/support/ome-artwork/.*$"
```

Comment:
Actually with this change, the form with no trailing slash i.e. '/site/support/ome-artwork' will not match and be redirected. We either need two redirects (`/?$` and `/.*$`) or to find a regexp way to combine them to capture all use cases

Reply:
Shall I add two for now?

Reply:
Unless you have the syntax for combining all use cases in one (i don't know it off-hand), two might be the fastest for now.

Reply:
I'm not sure why the rest of the forward slashes aren't escaped, but I'd try:

`"~\/site\/support\/ome-artwork(\/.*)?$"`

This says "ome-artwork" on it's own, or followed by at least `/` then zero-or-more of any other character.

i.e. the `/` is required if there will be any more characters after `ome-artwork`

Reply:
see https://regexr.com/3lpp9 cc @hflynn @sbesson

Reply:
i.e. you can leave out the `\` before the forward slashes if you like, as if it's working for the rest then it'll work for this: but generally one would escape them. (That does depend on the flavor of Regex mind)